src/test/java/constants/TestConstants.java

```diff
@@ -5,9 +5,20 @@
 public class TestConstants {
     public static final String RESOURCE_PATH = "src/main/resources/";
     public static final String YANDEX_MAIL_URL = ConfigUtils.getProperty("yandexmail.url");
-    public static final String YANDEX_MAIL_USERNAME = ConfigUtils.getProperty("yandexmail.username");
-    public static final String YANDEX_MAIL_PASSWORD = ConfigUtils.getProperty("yandexmail.password");
+    public static final String YANDEX_MAIL_USERNAME_1 = ConfigUtils.getProperty("yandexmail.username_1");
+    public static final String YANDEX_MAIL_PASSWORD_1 = ConfigUtils.getProperty("yandexmail.password_1");
+    public static final String YANDEX_MAIL_USERNAME_2 = ConfigUtils.getProperty("yandexmail.username_2");
+    public static final String YANDEX_MAIL_PASSWORD_2 = ConfigUtils.getProperty("yandexmail.password_2");
     public static final String CONFIG_PROPERTIES = "src/test/resources/config.properties";
     public static final String YANDEX_MAIL_INBOX_TITLE = "Inbox — Yandex Mail";
+    public static final String MULTISELECT_TEST_URL = "https://demo.seleniumeasy.com/basic-select-dropdown-demo.html";
```

Comment:
Keeping URLs in property files instead of doing it in classes - is a common practice

Reply:
Done